pkg/api/v1/routes/handlers.go

```diff
@@ -79,8 +79,54 @@ func (r *Routes) serverConditionCreate(c *gin.Context) (int, *v1types.ServerResp
 		}
 	}
 
+	return r.conditionCreate(otelCtx, c, kind, serverID)
+}
+
+func (r *Routes) serverEnroll(c *gin.Context) (int, *v1types.ServerResponse) {
```

Comment:
Can you include tests for this endpoint

Reply:
Thanks Jeol!
It sounds basically I am understanding the ticket correctly?
I will start adding tests and verifying it works in sandbox